about.html

```diff
@@ -421,11 +421,6 @@
     linkedin: https://www.linkedin.com/in/rpwagner
     gh_handle: rpwagner
     subproject: Jupyter Security
-  - name: Carol Willing
-    photo: CarolWilling.jpeg
-    affiliation: Noteable
-    gh_handle: willingc
-    subproject: Jupyter Standards
```

Comment:
Should the description for Paul's representation in line 392 be updated from `Jupyter Foundations` to `Jupyter Foundations and Standards`?

Reply:
:+1: Yeah you're right. Thanks, @krassowski! Updated.